Slicesy.PizzaOrder.WebApi/Repositories/Models/Pizza.cs

```diff
@@ -0,0 +1,27 @@
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Slicesy.PizzaOrder.WebApi.Repositories.Models
+{
+    [Table("Pizza")]
+    public class Pizza
+    {
+        [Key]
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        public long Id { get; set; }
+
+        [Required(ErrorMessage = "Pizzeria id is required")]
+        public long PizzeriaId { get; set; }
```

Comment:
Czy tak powinno się odwzorowywać klucze obce?

Reply:
tu nie ma odwzorowania jeszcze haha

Reply:
No to czy to pole powinno być na ten moment?

Reply:
tzn i tak bedzie trzeba je napisac so idk xd

Reply:
chyba ze dam todo albo zakomentuje xd